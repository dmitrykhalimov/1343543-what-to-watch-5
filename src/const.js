export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
