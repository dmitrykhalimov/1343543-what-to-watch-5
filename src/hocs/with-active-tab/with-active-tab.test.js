import React from "react";
import renderer from "react-test-renderer";
import withActiveTab from "./with-active-tab";
import {TEST_MOCKS} from "../../const";

const MockComponent = () => {

  return (
    <React.Fragment>
    </React.Fragment>
  );
};

const MockComponentWrapped = withActiveTab(MockComponent);

describe(`withActiveTab`, () => {
  it(`Should withActiveTab render correctly`, () => {
    const tree = renderer
      .create(
          <MockComponentWrapped
            film={TEST_MOCKS.film}
            comments={TEST_MOCKS.comments}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
