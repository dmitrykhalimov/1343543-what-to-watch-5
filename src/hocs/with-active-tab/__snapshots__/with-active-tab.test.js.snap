// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveTab Should withActiveTab render correctly 1`] = `null`;
